addons/towing/functions/fnc_startTow.sqf

```diff
@@ -30,6 +30,6 @@ _unit removeItem _ropeClass;
 
 GVAR(cancel) = false;
 GVAR(canAttach) = false;
+GVAR(onMouseButtonDownEHID) = ["MouseButtonDown", LINKFUNC(onMouseButtonDown)] call CBA_fnc_addDisplayHandler;
```

Comment:
Why add it and remove it every time? What was wrong with the old method?

Reply:
`- fix immediate rope attachment when click Attach tow rope menu if player is close to vehicle;`
see also the 1st commit in this PR 0394bcf1c12865a19108efe65f97575fdef56344 (actually reason of PR)